pac4j-core/src/test/java/org/pac4j/core/util/CommonHelperTests.java

```diff
@@ -146,6 +146,11 @@ public void testAreEqualsIgnoreCaseAndTrimFails() {
         assertFalse(CommonHelper.areEqualsIgnoreCaseAndTrim(VALUE, null));
         assertFalse(CommonHelper.areEqualsIgnoreCaseAndTrim(NAME, VALUE));
     }
+    
+    @Test
+    public void testClassPathPrefix() {
```

Comment:
It should, but nothing is perfect. I think there is a lack of tests here: we should have ones for all prefixes and for the use case with no prefix.